src/main/resources/assets/admin/common/js/ui/button/MenuButton.ts

```diff
@@ -125,19 +138,23 @@ export class MenuButton
         }
     }
 
-    isMinimized() {
+    isMinimized(): boolean {
         return this.hasClass('minimized');
     }
 
-    setEnabled(enable: boolean) {
+    setEnabled(enable: boolean): void {
         this.dropdownHandle.setEnabled(enable);
         this.actionButton.setEnabled(enable);
     }
 
-    setToggleMenuOnAction(value: boolean) {
+    setToggleMenuOnAction(value: boolean): void {
         this.toggleMenuOnAction = value;
     }
 
+    protected getDropdownPosition(): MenuButtonDropdownPos {
+        return MenuButtonDropdownPos.LEFT;
+    }
+
```

Comment:
This is a bit strange. Before the changes it used to be like this:

`let children = [this.actionButton, this.dropdownHandle, this.menu];`

with `dropdownHandle` being **to the right** of the `actionButton`. However, with this new method default position of the `dropdownHandle` is `to the left`. It's important that we don't break anything with these changes, in case MenuButton was used elsewhere.